runtime/doc/api.txt

```diff
@@ -438,6 +438,42 @@ Example: create a float with scratch buffer: >
     call nvim_win_set_option(win, 'winhl', 'Normal:MyHighlight')
 >
 
+==============================================================================
+Eextended marks						   *api-extended-marks*
+
+An extended mark (extmark) represents a buffer annotation that follows
+movements as the buffer changes. They could be used to represent cursors,
+folds, misspelled words, and anything else that needs to track a logical
+location in the buffer over time.
+
+Example:
+
+We will set an extmark at the first row and third column. As the API is zero-
+indexed, use row and column counts 0 and 2:
+
+`let g:mark_ns = nvim_create_namespace('myplugin')`
+`let g:mark_id = nvim_buf_set_extmark(0, g:mark_ns, 0, 0, 2)`
+
+Passing in id=0 creates a new mark and returns the id. we can look-up a mark
+by its id:
+
+`echo nvim_buf_get_extmark_by_id(0, g:mark_ns, g:mark_id)`
+=> [1, 0, 2]
```

Comment:
According to `nvim_buf_get_extmark_by_id`, the return value  should be `[0, 2]`.

Reply:
Thanks, updated.